sdk/dotnet/AppService/ActiveSlot.cs

```diff
@@ -9,42 +9,6 @@
 
 namespace Pulumi.Azure.AppService
 {
-    /// <summary>
```

Comment:
Looks like we're losing lots of examples again :( We should investigate this before releasing.

Reply:
And CI just failed on a dirty work dir...

Reply:
Seems like a local env issue with generating. Regenerated and pushed changes.